src/main/java/io/zold/api/TaxBeneficiaries.java

```diff
@@ -0,0 +1,54 @@
+/*
+ * The MIT License (MIT)
+ *
+ * Copyright (c) 2018 Yegor Bugayenko
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included
+ * in all copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NON-INFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+ * SOFTWARE.
+ */
+package io.zold.api;
+
+import java.util.Comparator;
+import org.cactoos.iterable.Filtered;
+import org.cactoos.iterable.IterableEnvelope;
+import org.cactoos.iterable.LengthOf;
+import org.cactoos.iterable.Sorted;
+
+/**
+ * {@link Remote} nodes that should receive taxes.
```

Comment:
@victornoel taxes are paid to only 1 node.

Reply:
@llorllale are you sure? See https://github.com/zold-io/zold/blob/master/lib/zold/commands/taxes.rb#L102
Basically, in there, if a wallet is in debt, say of 10 zold, the client pays 1 zold (the maximum to be paid according to the whitepaper) to the first node it choose, then do the same 9 other times for other nodes it choses.

The whitepaper doesn't say exactly this, it says that one node must be paid, but I suppose this is because it was expected that a wallet would never find itself with more than 1 zold of taxes to be in debt with. Which can happen actually.

Also, nothing prevent as a client, to decide to distribute its taxes to multiple nodes, or is there? This doesn't go against the protocol I think… there may be something to clarify with this…

Reply:
@victornoel You are right... I've opened zold-io/papers#49 and zold-io/zold#450. Thanks

Reply:
@llorllale ok, so I will leave it like that